msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+      1. Select **View** | **Run** in Visual Studio Code. In the **RUN AND DEBUG** drop down menu, select the option, **Launch Add-in Outlook Desktop (Edge Chromium)** (*not* any of the "Launch *App* ..." options), and then press F5. The project builds and a Webpack dev-server window opens. This process can take a couple of minutes. Eventually, Outlook desktop opens.
+      1. Open the **Inbox** *of your Microsoft 365 account identity* and open any message. A **Contoso Add-in** tab with two buttons appears on the **Home** ribbon (or the **Message** ribbon, if you open the message in its own window).
+      1. Select the **Show Taskpane** button and a task pane opens. Select the **Perform an action** button and a small notification appears near the top of the message.
+      1. To stop debugging and uninstall the add-in, select **Run** | **Stop Debugging** in Visual Studio Code. If the Webpack dev-server window doesn't close, open the Visual Studio Code **TERMINAL** in the root of the project and run `npm stop`.
+
+### Run the app and add-in locally at the same time
+
+You can sideload and run the app and the add-in simultaneously, but the debugger can't reliably attach when both are running. So to debug, run only one at a time. 
+
+To debug the app, see the last step of the **Prepare the Teams app project** section.
+
+To debug the add-in, see the last step of the **Edit the tooling configuration files** section.
+
+To see both the app and the add-in running at the same time, take the following steps.
+
+1. *First, make sure Outlook desktop is closed.*
+1. In Visual Studio Code, open the Teams Toolkit.
+1. In the **ACCOUNTS** section, verify that you're signed into Microsoft 365.
+1. Select **View** | **Run** in Visual Studio Code. In the **RUN AND DEBUG** drop down menu, select the option, **Launch App and Add-in Outlook Desktop (Edge Chromium)**, and then press F5. The project builds and a Webpack dev-server window opens to host the add-in. The tab app is hosted in the Visual Studio Code terminal. This process can take a couple of minutes. Eventually, both of the following things happen:
+
+    - Teams opens in a browser with a prompt to add your tab app. *If Teams has not opened by the time Outlook desktop opens, then automatic sideloading has failed.* You can manually sideload it to see both the app and the add-in running at the same time. For sideloading instructions, see [Upload your app in Teams](/microsoftteams/platform/concepts/deploy-and-publish/apps-upload). You'll find the manifest.zip file to upload at `C:\Users\{yourname}\AppData\Local\Temp`.
+    - Outlook desktop opens.
+
+1. In the Teams prompt, select **Add** and the tab opens.
+1. In Outlook, open the **Inbox** *of your Microsoft 365 account identity* and open any message. A **Contoso Add-in** tab with two buttons appears on the **Home** ribbon (or the **Message** ribbon, if you open the message in its own window).
```

Comment:
```suggestion
1. Go to Outlook.
1. Open the **Inbox** and open any message. 
    A **Contoso Add-in** tab with two buttons appears on the **Home** ribbon (or the **Message** ribbon, if you open the message in its own window).
```

Reply:
done, but it's important to keep the caution about which account to use. Devs typically have more than one in Outlook.